service/pom.xml

```diff
@@ -24,6 +24,13 @@
       <artifactId>dropwizard-core</artifactId>
       <version>${dropwizard.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>net.sf.json-lib</groupId>
+      <artifactId>json-lib</artifactId>
+      <version>2.4</version>
+      <classifier>jdk15</classifier>
+    </dependency>
```

Comment:
This might be related to Eric and I doing a manual install of the AlexaSkillKit in our local maven branch? And that's taking precedent over VoiceInterface? Because it SHOULD work just fine without this inclusion here.